Scarab/Models/ModSelect.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Scarab.ViewModels;
+using PropertyChanged.SourceGenerator;
+using System.ComponentModel;
+
+namespace Scarab.Models
+{
+    public partial class ModSelect : INotifyPropertyChanged
```

Comment:
Instead of creating a whole new model, you could just take in a list of mods that could be uninstalled and return the list of mods that are marked to be uninstalled

Reply:
I wanted to do that, but I just don't see an easy way to do that. I need to store checked values somewhere

Reply:
Actually my main problem is tagitem already exists for basically the same purpose.

https://github.com/TheMulhima/Scarab/blob/29d288d1043b3f62cbe49c9c17b0399985c2ab0f/Scarab/Models/TagItem.cs#L3

Maybe we could make that a template class

Reply:
It doesn't really do the same thing, but I guess we can make a generic solution for this
`public record SelectableItem<T> (T Item, string DisplayName, bool IsSelected);`
So then we can use DisplayName and IsSelected in the view and the Item in the VM for some other stuff

Reply:
Yea that's basically what I wanted.